### DISCUSSION TITLE
How to hide the NavBar Title / Menu / Button on Small Devices?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#2155] [Q&A] [answered]
Hi there

Disclaimer:  I am yet to upgrade to version 3.

In you example here: https://v2.daisyui.com/components/navbar/

![image](https://github.com/saadeghi/daisyui/assets/5799306/eb5e248b-6f61-4fc4-b1ee-ae0230b4c1e1)

How can I hide the nav bar items on small devices?  I want in one instance to just show the hamburger menu only and not the title.

Thanks a lot.

**Comment by saadeghi** [comment, marked as answer]
You can use Tailwind CSS utility classes to hide items in specific screen sizes
- `lg:hidden` hides and element on `lg` screens and larger
- `max-lg:hidden` hides an element on all screens smaller than `lg`
